chia/rpc/wallet_rpc_api.py

```diff
@@ -538,7 +539,7 @@ async def create_new_wallet(self, request: Dict):
                         uint64(request.get("fee", 0)),
                     )
 
```

Comment:
GH isn't letting me add a comment in some earlier code, but line 522 where we collect the `backup_dids` probably needs some thought re: bech32m conversion. Might be worth applying the conversion there now to avoid having some parts of the RPC using hex strings and others using bech32m.

Reply:
Done.